airflow/utils/db.py

```diff
@@ -1304,10 +1306,10 @@ def _dangling_against_task_instance(session, source_table, dag_run, task_instanc
         )
 
     return (
-        session.query(*[c.label(c.name) for c in source_table.c])
+        select(*[c.label(c.name) for c in source_table.c])
```

Comment:
Let’s also change this to `select(*(...))` (in the other planned PR). There are several of these around in this file too.

Reply:
@uranusjr Creating a tuple rather than a list is more efficient?

Reply:
This does not create an intermediate tuple but expands a generator directly. With `*[]` the generator is first realised into a list and then the list is expanded.